<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Suiteone" time="7" tests="0" failures="1" errors="0">
   <testsuite name="Suiteone" tests="0" failures="1" errors="0" time="7" timestamp="2018-03-19 12:11:12" hostname="AbdullahRashid - 10.6.226.103" id="Test Suites/Suiteone">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="/Users/abdullahrashid/Katalon Studio/Deal_Alert_UI_testing/Reports/Suiteone/20180319_121105"/>
         <property name="logFiles" value="/Users/abdullahrashid/Katalon Studio/Deal_Alert_UI_testing/Reports/Suiteone/20180319_121105/execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="AbdullahRashid - 10.6.226.103"/>
         <property name="hostAddress" value="10.6.226.103"/>
         <property name="os" value="Mac OS X 64bit"/>
         <property name="katalonVersion" value="5.3.1.1"/>
      </properties>
      <testcase name="Test Cases/Testing_Alert_First_page" classname="Test Cases/Testing_Alert_First_page" status="FAILED">
         <failure type="FAILED" message="Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)"/>
         <system-out><![CDATA[2018-03-19 12:11:13 - [TEST_CASE][FAILED] - Test Cases/Testing_Alert_First_page: Test Cases/Testing_Alert_First_page FAILED because (of) Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)

2018-03-19 12:11:14 - [TEST_STEP][FAILED] - openBrowser: Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)

2018-03-19 12:11:19 - [MESSAGE][FAILED] - Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)]]></system-out>
         <system-err><![CDATA[2018-03-19 12:11:13 - [TEST_CASE][FAILED] - Test Cases/Testing_Alert_First_page: Test Cases/Testing_Alert_First_page FAILED because (of) Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)]]></system-err>
      </testcase>
      <system-out><![CDATA[2018-03-19 12:11:12 - [TEST_SUITE][FAILED] - Suiteone: Test Cases/Testing_Alert_First_page FAILED because (of) Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)]]></system-out>
      <system-err><![CDATA[2018-03-19 12:11:12 - [TEST_SUITE][FAILED] - Suiteone: Test Cases/Testing_Alert_First_page FAILED because (of) Unable to open browser with url: '' (Root cause: org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:07:36.161Z'
System info: host: 'KRG-71f9-IFairooz.local', ip: '10.6.226.103', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.3', java.version: '1.8.0_102'
Driver info: driver.version: CGeckoDriver
remote stacktrace: stack backtrace:
   0:        0x10e2dea8e - backtrace::backtrace::trace::h69682bcb53decaf6
   1:        0x10e2deacc - backtrace::capture::Backtrace::new::hdc9d731a957304a6
   2:        0x10e20f434 - webdriver::error::WebDriverError::new::h930061353eac4c81
   3:        0x10e216dcc - geckodriver::marionette::MarionetteHandler::create_connection::hc09196342b760e35
   4:        0x10e1f67d4 - _$LT$webdriver..server..Dispatcher$LT$T$C$$u20$U$GT$$GT$::run::hfda25a6dc0b512aa
   5:        0x10e1ce645 - std::sys_common::backtrace::__rust_begin_short_backtrace::h95009c1d3a320838
   6:        0x10e1d611d - std::panicking::try::do_call::h8c5f07f1fc714fb2
   7:        0x10e38f9cc - __rust_maybe_catch_panic
   8:        0x10e1ebf55 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h6a96e09ff4d37bff
   9:        0x10e38bc3b - std::sys::imp::thread::Thread::new::thread_start::h823686b907c11c46
  10:     0x7fff762036c0 - _pthread_body
  11:     0x7fff7620356c - _pthread_start)]]></system-err>
   </testsuite>
</testsuites>
